src/evict/evict_page.c

```diff
@@ -44,6 +44,36 @@ __evict_exclusive(WT_SESSION_IMPL *session, WT_REF *ref)
     return (__wt_set_return(session, EBUSY));
 }
 
+/*
+ * __wt_page_evict_soon_check --
+ *     Check whether the page should be evicted urgently.
+ */
+bool
```

Comment:
This can be a inline function.

Reply:
Updated in [f663ce1](https://github.com/wiredtiger/wiredtiger/pull/7687/commits/f663ce11404822eedca1fd47d48e1a1ec9ff33b6)